src/sentry/api/endpoints/organization_events_facets_performance.py

```diff
@@ -0,0 +1,50 @@
+import sentry_sdk
+
+from collections import defaultdict
+from rest_framework.response import Response
+
+from sentry.api.bases import OrganizationEventsV2EndpointBase, NoProjects
+from sentry.snuba import discover
+from sentry import features, tagstore
+
+
+class OrganizationEventsFacetsPerformanceEndpoint(OrganizationEventsV2EndpointBase):
+    def has_feature(self, organization, request):
+        return features.has(
+            "organizations:performance-tag-explorer", organization, actor=request.user
+        )
+
+    def get(self, request, organization):
+        if not self.has_feature(organization, request):
+            return Response(status=404)
+
+        try:
+            params = self.get_snuba_params(request, organization)
+        except NoProjects:
+            return Response([])
+
+        aggregate_column = request.GET.get("aggregateColumn", "duration")
+
+        with sentry_sdk.start_span(op="discover.endpoint", description="discover_query"):
+            with self.handle_query_errors():
+                facets = discover.get_performance_facets(
+                    query=request.GET.get("query"),
+                    params=params,
+                    referrer="api.organization-events-facets-performance.top-tags",
+                    aggregate_column=aggregate_column,
+                )
+
+        with sentry_sdk.start_span(op="discover.endpoint", description="populate_results") as span:
+            span.set_data("facet_count", len(facets or []))
+            resp = defaultdict(lambda: {"key": "", "topValues": []})
```

Comment:
I was just relying on the existing code since it's all modelled after events_facets, probably going to leave this for now at least since I want the same serialization for testing.